pkg/virt-launcher/virtwrap/converter/converter.go

```diff
@@ -206,10 +206,12 @@ func Convert_v1_Disk_To_api_Disk(c *ConverterContext, diskDevice *v1.Disk, disk
 		}
 	}
 	disk.Driver = &api.DiskDriver{
-		Name:  "qemu",
-		Cache: string(diskDevice.Cache),
-		IO:    string(diskDevice.IO),
+		Name:        "qemu",
+		Cache:       string(diskDevice.Cache),
+		IO:          string(diskDevice.IO),
+		ErrorPolicy: "stop",
```

Comment:
somehow I remembered that `stop` is the default. but setting it explicitly is better, I think.

Reply:
hi @vladikr the default is `report` not `stop`, you can find a more detailed description in the issue https://github.com/kubevirt/kubevirt/issues/4799 and in [libvirt docs](https://libvirt.org/formatdomain.html). But to summarize the default value is report since libvirt 0.9.7.

Reply:
If we don't explicitly set to `stop` the VM is not paused on IO errors, and the errors are reported by the guest kernel.

Reply:
Great, thanks @alicefr